patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -7441,6 +7441,27 @@
  		int pickedProjectileId = -1;
  		if (PickAmmo_TryFindingSpecificMatches(sItem.type, item.type, out pickedProjectileId))
  			projToShoot = pickedProjectileId;
+@@ -41015,7 +_,8 @@
+ 			projToShoot = 117;
+ 
+ 		if (projToShoot == 42) {
++			/*
+-			if (item.type == 370) {
++			 if (item.type == 370) {
```

Comment:
Remove the space.

Reply:
Fixed. I'm not sure how I missed that one when I didn't miss the other space in the comment in WorldGen.cs.